vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -302,13 +302,13 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
     });
 
     const dotnetListVersionsRegistration = vscode.commands.registerCommand(`${commandPrefix}.${commandKeys.listVersions}`,
-        async (commandContext: IDotnetListVersionsContext | undefined, customWebWorker: WebRequestWorker | undefined): Promise<IDotnetListVersionsResult | undefined> =>
+        async (commandContext: IDotnetListVersionsContext | undefined, customWebWorker: WebRequestWorkerSingleton | undefined): Promise<IDotnetListVersionsResult | undefined> =>

```

Comment:
I haven't gotten to where this is defined yet, but I'm very optimistic based on the name